packages/vector_graphics/lib/src/listener.dart

```diff
@@ -363,6 +382,50 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
       _canvas.restore();
     }
   }
+
+  @override
+  void onImage(int imageId, int format, int width, int height, Uint8List data) {
+    // TODO(jonahwilliams): support other formats.
```

Comment:
Spec says conforming parsers need to support JPEG, PNG, and SVG.

Doing others is fine, but PNG nad SVG are the only ones I've actually seen in bug reports.

Reply:
Actually, that would be for the compiler, not for this - here I think it's fine if we only support png and raw, and otherwise inline svg.

Reply:
Alternatively we could convert to webP and maybe save more binary size...

Reply:
(it's fine to file this as an issue for now of course)

Reply:
I was actually thinking we could probably convert raw RGBA into PNG to save space too, but if we support webp everywhere that might be a reasonable target too

Reply:
Won't raw RGBA take more space though?

I guess it might compress better...  Would be interesting to experiment with.

Reply:
No, I mean take RGBA input from the SVG and convert it to another format

Reply:
Ahhh the input is typically base64 encoded png. I don't think raw RGBA is supported. Chrome probably supports BMPs or something but I've never seen it.

Reply:
Hmm, OK - lets stick with png only for now.